pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -897,6 +902,7 @@ public void start() throws PulsarServerException {
             }
 
             this.metricsGenerator = new MetricsGenerator(this);
+            this.openTelemetry = new PulsarBrokerOpenTelemetry(config);
```

Comment:
Is this always enabled? Shouldn't it be opt-in for now?

Reply:
Meaning that no instance would be created unless Otel is configured? Or is this by design?

Reply:
We have have this discussion later. I'll resolve the comment so that we can get the PR merged asap.